jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -515,7 +514,10 @@ def _extract_from_aperture(self, cube, uncert_cube, aperture,
                 # NOTE: just forcing these units for now!! this is in steradians and
                 # needs to be converted to the selected square angle unit but for now just
                 # force to correct units
-                aperture_area = self.cube.meta.get('PIXAR_SR', 1.0) * sq_angle_unit
+                if sq_angle_unit == u.sr:
+                    aperture_area = self.cube.meta.get('PIXAR_SR', 1.0) * sq_angle_unit
+                else:
+                    aperture_area = 1 * sq_angle_unit
```

Comment:
This is the spectral extraction bug. One should ignore PIXAR_SR if the solid angle is not steradian.

Reply:
Yes. That's why I had to fix it even though it was out of scope. Otherwise the value for this combo is wrong in `test_basic_unit_conversions`.